### ISSUE TITLE
[Feature Request] execute code cell

### ISSUE THREAD

**Issue by kikyous** [HappenApps/Quiver#599]

**Comment by CreativeCode** [comment]
I would really appreciate this as well...But I think I read somewhere that the developers see Quiver as a 'static' notebook and that they do not want to head into this direction (?).... I guess what you are looking for might be done with jupyter. (I wish Quiver would allow editing jupyter notebooks though...)

**Comment by ghost** [comment]
I think the most useful thing would be for Quiver to use the Jupyter notebook format as its native file format - so it could open Jupyter notebooks and vice-versa. But built-in import/export to/from Jupyter would be the next best thing. As for executing the code, that would essentially turn Quiver into Jupyter and would make it much more heavyweight (and less likely to work on iOS); I think the current approach makes more sense.

**Comment by the-c0d3r** [comment]
+1 to `built-in import/export to/from Jupyter`. So I can maybe write it in Quiver, export to Jypter, and execute on it. But then IMHO, it kind of defeat the point of using Quiver in the first place though.

**Comment by CreativeCode** [comment]
I completely agree with @geekraver. It would be ideal if Quiver would adopt (or maybe extend?) the Jupyter notebook format and consequently would support editing (or at least importing/exporting) Jupyter notebooks. 

There are many situations where it makes sense to edit a Jupyter notebook with Quiver.  Primarily on iOS - where no good way of editing Jupyter notebooks exists at all (please correct me if I am wrong). But also on OS X, simply because the Quiver interface and user experience is great.... For instance, I would certainly suggest Quiver to my students for note-taking during programming classes (many students use an iPad these days), and ask them to continue with Jupyter in the Lab sessions/computer rooms later on.

**Comment by ylian** [comment]
I agree with everything said here, but I have to draw a line somewhere to make sure Quiver doesn't grow into a monster that tries to be everything for everyone. I think Quiver and Jupyter are two different products, and have different focuses. I like Jupyter a lot and see it as an exploratory tool more than a documentation tool. I would like Quiver to focus on being a great, static notebook, and there is still tons of things to do in the core functionalities.

**Comment by craighurley** [comment]
> make sure Quiver doesn't grow into a monster that tries to be everything for everyone

You are a wise man.  Please don't listen to requests for features like this one.  Executing code from a **note taking** app is nonsensical.

Evernote was an ok app for storing notes, then stagnated while they added junk features (e.g. chat) -- I'm pretty sure that's why you have many people now using Quiver.

**Comment by lisah2u** [comment]
Actually executing code from note-taking is an interesting modern idea. See this thread by Stephen Wolfram on Computational Essays. 
http://blog.stephenwolfram.com/2017/11/what-is-a-computational-essay/

Perhaps, the bridge is enabling much better import / export functionality to Jupyter, GitHub, Rmd, etc. Or extending to Atom, for example. Quiver does seem to fill a gap in computational and mathematical note-taking, but it needs better integration across ecosystems. 

I'm disappointed that I can't use Quiver to open an existing markdown in GitHub pages so I can easily improve on documents referencing images and other documents (to include JS) in that repository.

**Comment by Bulletninja** [comment]
Perhaps this can be optional, as in a plugin system?
